draft-foudil-securitytxt.md

```diff
@@ -257,11 +260,12 @@ implementors MUST ignore any fields they do not explicitly support.
 # File Format Description
 
 The expected file format of the security.txt file is plain text as defined
-in section 4.1.3 of {{!RFC2046}} and encoded in UTF-8.
+in section 4.1.3 of {{!RFC2046}} and follow {{!RFC5198}}.
```

Comment:
I would recommend to follow the convention of RFC 7252:
https://tools.ietf.org/html/rfc7252#section-3.2

and would suggest the following wording: **"and is encoded using UTF-8 [RFC3629] in Net-Unicode form [RFC5198]"**